server/controllers/finance/reports/invoices/receipt.handlebars

```diff
@@ -159,6 +159,15 @@
             </td>
           </tr>
         {{/if}}
+
+        {{!-- is convention --}}
+        {{#if recipient.is_convention}}
```

Comment:
I think `is_convention` is a poor name.  It would be better to call it `hasConventionCoverage` or `is_covered` or something.